src/turn_integrator.py

```diff
@@ -25,22 +26,42 @@ def turn_credentials(line_name):
 
     return eval_credentials(config_json)
 
-### Obtain a new auth token from Turn.io
-### TODO Not tested yet
-def obtain_auth_token(username, password):
-    u_pass = HTTPBasicAuth(username, password)
-    return requests.post('https://whatsapp.turn.io/v1/users/login', auth=u_pass)
+### CONTACTS
+## Obtain a contact profile. See documentation here:
+## https://whatsapp.turn.io/docs/api/contacts#retrieve-a-contact-profile
+def obtain_contact_profile(msisdn: str, line_name: str) -> requests.Response:
+    auth_headers = {
+        'Authorization': f'Bearer {turn_credentials(line_name)}',
+        'Accept': 'application/vnd.v1+json'
+    }
+
+    response = requests.get(f'https://whatsapp.turn.io/v1/contacts/{msisdn}/profile', headers=auth_headers)
+    print(response.text)
+    return response
+
+## Update a contact profile. Supply only the fields that need updating. See documentation here:
+## https://whatsapp.turn.io/docs/api/contacts#update-a-contact-profile
+def update_contact_profile(msisdn: str, line_name: str, profile_data: json) -> requests.Response:
+    auth_headers = {
+        'Authorization': f'Bearer {turn_credentials(line_name)}',
+        'Accept': 'application/vnd.v1+json'
+    }
+
+    response = requests.patch(f'https://whatsapp.turn.io/v1/contacts/{msisdn}/profile', headers=auth_headers, json=profile_data)
```

Comment:
We should probably keep these urls in a .env file that we can manage? Probably a .env that can be commited to the repo?